### DISCUSSION TITLE
Implement the Compobox

### DISCUSSION THREAD

**Discussion by t1m4lc** [rgossiaux/svelte-headlessui#104] [Ideas]
It's probably like the listbox but with autocomplete behaviour. 

https://headlessui.dev/react/combobox

**Comment by multiplehats** [comment]
Only thing that requires this is to merge this PR: https://github.com/rgossiaux/svelte-headlessui/pull/126

Seems like it's good to go.

**Comment by ryou90** [comment]
HI folks,
there is already a implementation https://github.com/NoelOConnell/svelte-headlessui/tree/combobox-component
You can also find the package on https://www.npmjs.com/package/svelte-headlessui-combobox-temporary

There exists also a really new kind of implementation here: https://github.com/CaptainCodeman/svelte-headlessui
